src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
+    public double add(double a1, double a2){
```

Comment:
a1, a2 is acceptable naming, but has it's drawbacks:
It's more complex because we use TWO alphanumeric systems:
a, b, c, d..
and
1, 2...

We can reduce complexity just by using mathematic notation for arithmetic operations:
a and b:
```
  public double subtract(double a, double b){
        return a - b;
    }
```
It's a little bit more easier to read